src/node_config_file.cc

```diff
@@ -0,0 +1,166 @@
+#include "node_config_file.h"
+#include "debug_utils-inl.h"
+#include "simdjson.h"
+
+#include <string>
+
+namespace node {
+
+std::optional<std::string> ConfigReader::GetDataFromArgs(
```

Comment:
No need to copy string here

```suggestion
std::optional<std::string_view> ConfigReader::GetDataFromArgs(
```